frontend/src/views/admin/matching/applicant-view/grid/modals.tsx

```diff
@@ -67,7 +67,7 @@ export function AdjustHourModal({
                 <Modal.Title>Update Hours</Modal.Title>
             </Modal.Header>
             <Modal.Body>
-                <Form>
+                <Form onSubmit={(e) => e.preventDefault()}>
```

Comment:
This is not the right solution. Other dialogs do not do this. You probably shouldn't be using `Form` controls at all.